terraform-modules/cloud-function/test/test.tf

```diff
@@ -0,0 +1,40 @@
+module "test_cloud_function" {
```

Comment:
Just confirming, does this test do a real apply? Seems like it's just a validate but want  to check.

Reply:
It is not a real apply--I suppose if you ran terraform apply in that directory it would, but the CI does not. The only way it would do anything would be if you applied it locally, because the providers would need to hook into your local machine's auth to do anything.

Reply:
Great just wanted to verify. Doing infrastructure testing where you actually create the resources for real is a tricky subject, that is complex and needs to be done carefully. 

Non issue here though 👍🏻